pkgs/applications/misc/deadd-notification-center/default.nix

```diff
@@ -1,44 +1,49 @@
-{ lib, stdenv
-, fetchurl
+{ lib
+, stdenv
+, fetchFromGitHub
 , autoPatchelfHook
+, wrapGAppsHook
+, hicolor-icon-theme
 , gtk3
 , gobject-introspection
 , libxml2
 }:
-
 let
-  version = "1.7.2";
-
-  dbusService = fetchurl {
-    url = "https://github.com/phuhl/linux_notification_center/raw/${version}/com.ph-uhl.deadd.notification.service.in";
-    sha256 = "0jvmi1c98hm8x1x7kw9ws0nbf4y56yy44c3bqm6rjj4lrm89l83s";
-  };
-in
-stdenv.mkDerivation rec {
-  inherit version;
   pname = "deadd-notification-center";
-
-  src = fetchurl {
-    url = "https://github.com/phuhl/linux_notification_center/releases/download/${version}/${pname}";
-    sha256 = "13f15slkjiw2n5dnqj13dprhqm3nf1k11jqaqda379yhgygyp9zm";
+  version = "1.7.3";
+in
```

Comment:
Can you remove this let in? It is not really required here.

Reply:
made it a rec set instead